Fixes #110

Fonta1n3 opened the pull request:
fix: update beta label for testing node for App Store purposes

fix: fixes #110, if no recipient is specified do not assume addresses are change, this happens when a psbt is uploaded externally instead of created via the out tab

Fonta1n3 committed:
fix: update beta lable for testing node

fix: if no recipient is specified do not assume addresses are change, this happens when a psbt is uploaded externally instead of created via the out tab

Fonta1n3 committed:
fix: fix a bug where updating the node creds would fail via QR scan, fixes a bug where adding a Quick Connect QR via the home screen could have resulted in an error

ChristopherA commented:
I tried building this, and I get:

```
Swift Compiler Error Group
/Users/christophera/Library/Developer/Xcode/DerivedData/GordianWallet-cxxdijbptomjasdjvrujaghbtfvy/Build/Products/Debug-iphoneos/SSKR.framework/Modules/SSKR.swiftmodule/arm64-apple-ios.swiftinterface:4:8: No such module 'CSSKR'
/Users/christophera/projects/github/BlockchainCommons/GordianWallet-iOS/XCode/GordianWallet/View Controllers/Wallets/Wallet Recovery/WalletRecoverViewController.swift:11:8: Failed to build module 'SSKR' from its module interface; the compiler that produced it, 'Apple Swift version 5.2.2 (swiftlang-1103.0.32.6 clang-1103.0.32.51)', may have used features that aren't supported by this compiler, 'Apple Swift version 5.3 (swiftlang-1200.0.29.2 clang-1200.0.30.1)'
```
and
```
Warning Group
Wallets.xprvs is using a nil or insecure value transformer. Please switch to NSSecureUnarchiveFromDataTransformerName or a custom NSValueTransformer subclass of NSSecureUnarchiveFromDataTransformer [2]
```

Fonta1n3 committed:
chore: update frameworks

Fonta1n3 commented:
As per #121 I have moved this PR into draft status. 

I will quickly fix #121 but the codebase will need much more testing as so much was changed (and not tested) in [https://github.com/BlockchainCommons/GordianWallet-iOS/pull/111/commits/fd31756c98902e3021191f9b5b87a19e7222b57a](https://github.com/BlockchainCommons/GordianWallet-iOS/pull/111/commits/fd31756c98902e3021191f9b5b87a19e7222b57a).

Fonta1n3 committed:
chore: updates libwally

fix: bug where entropy was not being used correctly after API change to libwallyswift

Fonta1n3 committed:
fix: export address was showing the libwally data structure instead of actual address string

Fonta1n3 committed:
fix: do not show spendable balance if unconfirmed

Fonta1n3 committed:
feat: start adding UI for exporting a cosigner

fix: bug where constraints were not correct for macOS on home screen
fix: do not auto resign Tor on macOS

Fonta1n3 committed:
feat: add ability to export cosigner in the form of ur:crypto-hdkey from account tools (if seed words are present)

Fonta1n3 committed:
feat: add logic for converting an account to ur:crypto-output in Backup Info

Fonta1n3 committed:
feat: add switch action to ur

Fonta1n3 committed:
fix: remove unused script type from descriptor parser

feat: WIP continue adding crypto-output encoding support

feat: convert all items in "backup info" to UR by default

Fonta1n3 committed:
minor

Fonta1n3 committed:
fix: bug where bip48 nested segwit derivation was not parsed correctly

fix: crypto-output export

Fonta1n3 committed:
fix: crash when exporting a cosigner where no seed is present

fix: qr scanning on macos now works

Fonta1n3 committed:
fix: crypto-output encoding

Fonta1n3 committed:
fix: account lifehashes should now match GCS

Fonta1n3 committed:
fix: show lifehash of address for invoice creation

shannona commented:
Merging to bring us up to date with Testflight Gordian releases.
